### DISCUSSION TITLE
大佬救命，配置这里没有反应

### DISCUSSION THREAD

**Discussion by sbtv587** [mikumifa/biliTickerBuy#183] [提问专区]
![微信图片_20240628122309](https://github.com/mikumifa/biliTickerBuy/assets/174082278/b6173338-8c91-43e5-8283-e85c99ff754b)

**Comment by sbtv587** [comment]
@mikumifa

**Comment by mikumifa** [comment]
你可能没登录

**Comment by YourComputer011** [comment]
这里会卡一会，大概半分钟，之后就会弹出登陆界面

**Comment by mikumifa** [comment]
那就是要登录了

**Reply by sbtv587** [reply]
佬，怎么登陆啊

**Reply by Asuna380** [reply]
极验 GeeTest 验证失败是什么意思啊大佬QAQ
